wiki/Client/Quick_play/en.md

```diff
@@ -0,0 +1,66 @@
+---
+tags:
+  - quickplay
+  - matchmaking
+  - match making
+  - multiplayer
+  - MMR
+  - rating
+---
+
+# Quick play
+
+**Quick play** is a queue-based multiplayer mode introduced to [osu!(lazer)](/wiki/Client/Release_stream/Lazer) on October 29, 2025 ([release](https://github.com/ppy/osu/releases/tag/2025.1029.1-lazer)). It is the first queue-based multiplayer system introduced to the game.
+
+*Note: osu!mania 4K and 7K are split into separate queues and are thus referred to as separate modes.*
+
+## Interface
+
+Users may access quick play by navigating to `play > multi > quick play` via the main menu. The traditional multiplayer lobby interface is accessible via the `lounge` option.
+
+![](img/quick-play-main-menu.png "Main menu interface")
```

Comment:
these are missing alt text (in square brackets). can be the same as title text/caption (in quotes)